### PR TITLE
[CVSRP-4161] - Added new PreviousFurloughPeriods page

### PR DESCRIPTION
**Pull request by BackRowOla**

### CODE REVIEW

**Comment by BackRowOla** [comment]
Screenshot 2021-02-05 at 12 21 49

**Comment by BackRowOla** [comment]
Page not wired up but view renders

File: conf/messages.en [hunk 1]
```diff
@@ -878,3 +878,8 @@ furloughInLastTaxYear.error.required = Select one of the options below
 calculationUnsupported.heading = This calculation cannot be supported at the moment
 calculationUnsupported.p1 = You can do this calculation manually.
 calculationUnsupported.startAgain = Start a new calculation
+
+previousFurloughPeriods.title = Has this employee been furloughed more than once since 1 November 2020?
+previousFurloughPeriods.heading = Has this employee been furloughed more than once since 1 November 2020?
+previousFurloughPeriods.checkYourAnswersLabel = previousFurloughPeriods
```
Comment (peckoverHMRC) [inline conf/messages.en:884]: Is this used?

File: test/navigation/NavigatorSpecWithApplication.scala [hunk 2]
```diff
@@ -523,10 +523,27 @@ class NavigatorSpecWithApplication extends SpecBaseControllerSpecs with CoreTest
         navigator.nextPage(
           EmployeeStartDatePage,
           emptyUserAnswers
-            .withEmployeeStartDate("2019,8,1")
+            .withEmployeeStartDate("2020,3,20")
             .withClaimPeriodStart("2020,11,1")
         ) mustBe routes.PayDateController.onPageLoad(1)
 
+        println("test for previous furluogh periods - currently not working")
```
Comment (peckoverHMRC) [inline test/navigation/NavigatorSpecWithApplication.scala:530]: :println:

File: test/navigation/NavigatorSpecWithApplication.scala [hunk 3]
```diff
@@ -523,10 +523,27 @@ class NavigatorSpecWithApplication extends SpecBaseControllerSpecs with CoreTest
...
+        navigator.nextPage(
+          EmployeeStartDatePage,
+          emptyUserAnswers
+            .withEmployeeStartDate("2020,4,1")
+            .withFurloughStartDate("2020,11,10")
+            .withClaimPeriodStart("2020,11,1")
+        ) mustBe routes.PreviousFurloughPeriodsController.onPageLoad()
+
+        println("test for RTI submission page")
```
Comment (peckoverHMRC) [inline test/navigation/NavigatorSpecWithApplication.scala:539]: :println:

**Review by peckoverHMRC — CHANGES_REQUESTED** [review]

**Comment by peckoverHMRC** [comment]
all looks good to me. Just needs to be hooked up with CVSRP-4159 when its finished

**Comment by MJCallahanPage** [comment]
To be merged as part of CVSRP-4170

**Pull request closed without merging**